src/components/IconDropdown/_icon-dropdown.scss

```diff
@@ -0,0 +1,152 @@
+@import '~carbon-components/scss/components/dropdown/dropdown';
+
+@import '../../globals/vars';
+
+.#{$iot-prefix}--dropdown__image-button:hover {
+  border-color: transparent;
+
+  ::before {
+    opacity: 0;
+  }
+}
+
+.#{$iot-prefix}--icon-dropdown__footer {
+  @include box-shadow();
```

Comment:
I think the box shadow should be applied uniformly so that the box shadow is present with or without the footer. Currently it sort of flashes on and off. This capture isn't the best but you can see the shadow flashing on and off on hover.
![hover shadow](https://user-images.githubusercontent.com/3360588/98672987-4b0afd80-231c-11eb-96df-cb0ed1342c92.gif)